sdk/react/utils.js

```diff
@@ -28,6 +28,12 @@ var Utils = {
   },
 
   shouldShowLandscape: function(width, height) {
+    if (isNaN(width) || isNaN(height) ||
+        width === null || height === null ||
+        width < 0 || height < 0) {
+      return null;
```

Comment:
I added null, because if one of those conditions is met, it means that there's something wrong with the parameters. Returning false here, is telling that it shouldShowLandscape == false, but logically that's not true, because by returning null, I'm telling parameters are bad. 

But if we don't want to think about this problem, we may just return false, but if someone passes a bad parameter to this function, it may be troublesome to find the problem is here.